privacy.adoc

```diff
@@ -351,7 +351,7 @@ can only increase with layers on top, but that doesn't seem to be the
 case for privacy, which can be improved or made worse arbitrarily in
 layers on top. Why is that? Any layer on top of Bitcoin, as explained
 in <<_layered_scaling,Layered scaling>>, must use on-chain
-transactions now and then, otherwise, it wouldn't be "on top of
+transactions evrey now and then, otherwise it wouldn't be "on top of
```

Comment:
every

Or maybe we should use the word "occasionally" instead of "every now and then"? Layer's on top generally want to use the blockchain as little as possible.